Reference qvalue rule from semantics (#1683)
httpwg/http-extensions: draft-ietf-httpbis-digest-headers.md

   Want-Digest = 1#want-digest-value
   Want-Content-Digest = 1#want-digest-value
   want-digest-value = digest-algorithm [ ";" "q" "=" qvalue]
~~~

> `qvalue` (see Section 12.4.2 of {{SEMANTICS}}) indicates the sender's preference
> from highest (1) to lowest (0.001). If a digest-algorithm is not accompanied by a
> qvalue, it is treated as if its associated qvalue were 1.

Author: ```suggestion
`qvalue` indicates the sender's digest-algorithm's preferences. Section 12.4.2 of {{SEMANTICS}}) describes how endpoints use `qvalue`.
```

> A qvalue of 0 indicates that a sender does not accept the associated
> digest-algorithm.

Author: ```suggestion
```

> digest-algorithm.

Author: @MikeBishop integrated @LPardue 's comment. PTAL and thanks for the feedback!

   Want-Digest = 1#want-digest-value
   Want-Content-Digest = 1#want-digest-value
   want-digest-value = digest-algorithm [ ";" "q" "=" qvalue]
~~~

> `qvalue` indicates the sender's digest-algorithm's preferences. Section 12.4.2 of {{SEMANTICS}}) describes how endpoints use `qvalue`.

Author: ```suggestion
`qvalue` indicates the sender's digest-algorithm's preferences.
Section 12.4.2 of {{SEMANTICS}}) describes  `qvalue` usage and semantics.
```

   Want-Digest = 1#want-digest-value
   Want-Content-Digest = 1#want-digest-value
   want-digest-value = digest-algorithm [ ";" "q" "=" qvalue]
~~~

> `qvalue` indicates the sender's digest-algorithm's preferences.

Reviewer: ```suggestion
`qvalue` indicates the sender's digest-algorithm preferences.
```
They are the sender's preferences about algorithms, not the preferences of the algorithms themselves.

Reviewer 2: ahh such a bad use of an `'s` by me, the shame
